### DISCUSSION TITLE
UI inconsistencies

### DISCUSSION THREAD

**Discussion by aggKartik** [sandboxnu/office-hours#587] [General]
Hey everyone

I'm Kartik and I'm a TA for OOD. I've enjoyed using the app so far and it's made office hours much easier. 

I noticed a few UI inconsistencies / bugs that I wanted to bring up:

1. On a widescreen display like mine (3440 x 1440), the settings page alignment looks a little off
![Capture2](https://user-images.githubusercontent.com/24238951/107810112-29b21700-6d3a-11eb-8d65-acc4809845b0.PNG)

2. Is the padding of the active symbol missing next to the name?
![Capture](https://user-images.githubusercontent.com/24238951/107810114-2a4aad80-6d3a-11eb-8b4b-d82c4d89cf73.PNG)

Again, super minor stuff, but just noticed them so I thought I'd bring it up!

**Comment by willstenzel** [comment]
Hi @aggKartik, thanks for letting us know, the feedback is always super helpful.

I've created issues for both these UI inconsistencies (#588 and #589) so we'll make sure to get them fixed.
